src/MizarData/BaselineAndComparisonHelper.cpp

```diff
@@ -8,53 +8,57 @@
 #include <absl/container/flat_hash_set.h>
 #include <stdint.h>
 
+#include <string>
+
 namespace orbit_mizar_data {
 
 template <typename K, typename V>
 [[nodiscard]] static absl::flat_hash_set<V> ValueSet(const absl::flat_hash_map<K, V>& map) {
   absl::flat_hash_set<V> result;
   std::transform(std::begin(map), std::end(map), std::inserter(result, std::begin(result)),
-                 [](const std::pair<K, V> pair) { return pair.second; });
+                 std::mem_fn(&std::pair<std::add_const_t<K>, V>::second));
```

Comment:
(pers pref) I think the original version was easier to read/understand.. what are the benefits of the new version?

Reply:
> what are the benefits of the new version?
I had fun writing it. 

I guess, you're right and it should be lambda here.

Reply:
fwiw, I had fun trying to understand it